bundle/src/main/java/com/adobe/acs/commons/reports/models/package-info.java

```diff
@@ -15,7 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-@org.osgi.annotation.versioning.Version("1.2.0")
+@org.osgi.annotation.versioning.Version("1.2.1")
```

Comment:
```suggestion
@org.osgi.annotation.versioning.Version("1.2.0")
```

Why does this require a version bump to 1.21? Im not seeing where the public API is changed in this package. Am i missing something?

Reply:
Micro version adjustments are required for almost all changes. You should never modify bytecode in a package without adjusting at least micro version.